brainstormforce/astra
# AST-3244 - Admin Astra Branding logo design tweaks

### Description
- Admin logo size updated to 20px
- Logo updated as per provided figma design

### Screenshots


### Types of changes
- non-breaking change which fixes an issue

### How has this been tested?
- Check with default Astra icon (for admin left panel, top admin bar on frontend for CL & PH)
- Admin left panel Astra icon size 18px removed and make it default 20px (as per last requirement)
- Check normal, hover states of the both location icons

### Checklist:
